CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Getting Started
 
-Install Node.js if you haven't already and run:
+Install [Node.js](https://nodejs.org/) if you haven't already, and run to install project dependencies:
```

Comment:
Maybe we could rephrase the sentence like 

Install [Node.js](https://nodejs.org/) if you haven't already, and run the following command to install project dependencies: